adapters/oidc/js/src/keycloak.js

```diff
@@ -1299,13 +1299,20 @@ function Keycloak (config) {
 
     function check3pCookiesSupported() {
         var promise = createPromise();
+       
+        // If init() has been called once
+        if(loginIframe.iframe3p){
+            promise.setSuccess();
```

Comment:
It would be better to throw an exception if `init()` is called more than once. It should only be called only once.

Also, let's use a separate boolean property such as `initCalled` to keep track of this rather than hijacking `iframe3p`.